rfd/0067-desktop-access-file-system-sharing.md

```diff
@@ -443,6 +443,36 @@ This message is sent by the client to the server in response to a `Shared Direct
 `fso_list` is a list of `fso` objects representing all the files and directories in the directory specified by the `path` variable in the originating
 `Shared Directory List Request`.
 
+#### 33 - Shared Directory Truncate Request
+
+```
+| message type (33) | completion_id uint32 | directory_id uint32 | path_length uint32 | path []byte | end_of_file uint32 |
+```
+
+This message is sent by the server to the client to request a file be truncated to `end_of_file` bytes.
```

Comment:
Can we be more specific w.r.t what is the server and what is the client.

In standard RDP it's pretty clear, but our architecture introduces some ambiguity - the w_d_s is an RDP client, but can also be considered the server to our web-based client.

Reply:
I added a note on the terminology in 197a123e8788b121e9131878a509f7f2452c7e92